tell/execute_forward.py

```diff
@@ -52,7 +52,7 @@ def extract_ba_code(filename):
     :param filename: str -> name of the MLP output file
     :return: ba_code: str -> alphanumeric code of the balancing authority
     """
-    ba_code = filename[filename.rindex('\\') + 1:].rstrip('_mlp_predictions.csv')
+    ba_code = filename[filename.rindex(os.sep) + 1:].rstrip('_mlp_predictions.csv')
```

Comment:
the backslash is very particular to Windows systems -- `os.sep` resolves to the appropriate path separator in the user's environment. but, if there's a way to do this without relying on the file name, it could be better long term

Reply:
I agree loading by file name is not very robust.  I will look into loading without it